spec/services/allocation_service_spec.rb

```diff
@@ -11,7 +11,7 @@
   describe '#allocate_secondary', :queueing do
     let(:kath_id) { 485_637 }
     let(:ross_id) { 485_926 }
-    let(:nomis_offender_id) { 'G7806VO' }
+    let(:nomis_offender_id) { 'G4273GI' }
```

Comment:
What made this spec change along with the change we're trying to make?

Reply:
We needed an offender with the right combination of release dates for this set of specs to work properly